test/chrome-launcher-test.ts

```diff
@@ -133,6 +133,11 @@ describe('Launcher', () => {
     assert.ok(!chromeFlags.includes('--disable-extensions'));
   });
 
+  it('searches for available installations', async () => {
+    const installations = Launcher.getInstallations();
+    assert.ok(Array.isArray(installations));
```

Comment:
this test case is super weak sauce. very much open to, and would appreciate, suggestions for how to better test this feature.

as it stands, this coverage at least ensures that the method exists, is callable, and returns an array --which is better than nothing, but fails to ensure that the appropriate resolution logic is actually used.

I struggled with how to go about testing this method for a few reasons:
1. the project doesn't appear to include any module mocking dependencies and I didn't feel comfortable pulling one in without first reaching consensus that I should
2. all of the heavy lifting is done by chrome-finder, while platform resolution logic is handled by utils; short of asserting that `getPlatform()` and `chromeFinder[]()` are invoked (which likely requires a mocking lib ^^), I'm not sure what else to assert.